bolt-sidecar/src/test_util.rs

```diff
@@ -1,3 +1,4 @@
+use crate::ChainConfig;
```

Comment:
can you bring this import down closer to his frens after `use crate::`? it's sad and alone here